samples/bluetooth/peripheral_hids_mouse/README.rst

```diff
@@ -1,24 +1,129 @@
-.. _peripheral_hids:
+.. _peripheral_hids_mouse:
 
-Bluetooth: Peripheral HIDs mouse
+Bluetooth: Peripheral HIDS mouse
 ################################
 
+The Peripheral HIDS mouse sample demonstrates how to use the :ref:`hids_readme` to implement a mouse input device that you can connect to your computer.
+
 Overview
 ********
 
-Similar to the :ref:`Peripheral <ble_peripheral>` sample, except that this
-application specifically exposes the HID GATT Service. The report map used is
-for a generic mouse.
+The sample uses the buttons on a development board to simulate the movement of a mouse.
+The four buttons simulate movement to the left, upward, to the right, and downward, respectively.
+Mouse clicks are not simulated.
+
+This sample is similar to Zephyr's :ref:`zephyr:ble_peripheral` sample, except that it specifically exposes the HID GATT Service.
+It uses a report map for a generic mouse.
 
 
 Requirements
 ************
 
-* A board with BLE support
+* One of the following development boards:
+
+  * nRF9160 DK board (PCA10090)
+  * nRF52840 Development Kit board (PCA10056)
+  * nRF52 Development Kit board (PCA10040)
+  * nRF51 Development Kit board (PCA10028)
+
+User interface
+**************
+
+Button 1:
+   Simulate moving the mouse pointer 5 pixels to the left.
+
+Button 2:
+   Simulate moving the mouse pointer 5 pixels upward.
+
+Button 3:
+   Simulate moving the mouse pointer 5 pixels to the right.
+
+Button 4:
+   Simulate moving the mouse pointer 5 pixels downward.
 
-Building and Running
+
+
+Building and running
 ********************
 
 This sample can be found under :file:`samples/bluetooth/peripheral_hids_mouse` in the |NCS| folder structure.
 
 See :ref:`gs_programming` for information about how to build and program the application.
+
+Testing
+=======
+
+After programming the sample to your board, you can test it either by connecting the board as a mouse device to a Microsoft Windows computer or by connecting to it with `nRF Connect for Desktop`_.
+
+Testing with a Microsoft Windows computer
+-----------------------------------------
+
+To test with a Microsoft Windows computer that has a Bluetooth radio, complete the following steps:
+
+1. Power on your development board.
+#. On your Windows computer, search for Bluetooth devices and connect to the device named "NCS HIDS mouse".
+#. Push Button 1 on the board.
+   Observe that the mouse pointer on the computer moves to the left.
+#. Push Button 2.
+   Observe that the mouse pointer on the computer moves upward.
+#. Push Button 3.
+   Observe that the mouse pointer on the computer moves to the right.
+#. Push Button 4.
+   Observe that the mouse pointer on the computer moves downward.
+#. Disconnect the computer from the device by removing the device from the computer's devices list.
+
```

Comment:
The nRF5 SDK documentation has information about Testing Swift Pair in addition - I assume that is not applicable here?
https://www.nordicsemi.com/DocLib/Content/SDK_Doc/nRF5_SDK/v15-2-0/ble_sdk_app_hids_mouse